testregistry.textproto

```diff
@@ -408,6 +408,12 @@ test: {
   readme: " "
```

Comment:
Probably PF-1.5 should be a TUN-*.* ID test because it sounds like from the description that it is based on /interfaces/interface/tunnel configuration